1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,151 @@
+Feature: User edits Project
+    In order to document a project 
+    As a logged in User
+    I want to edit existing and create new projects
+
+    @xxx
+    Scenario Outline: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        # Save button is removed as Project API demands a PM
```

Comment:
Nå vil vi vel ha samme lagre-knapp her som på Participants-tab?

Reply:
Eg sjekker med Tor-Arne. Om API/datamodel tillater det, så kan vi gjerne opne for det.

Reply:
Tor-Arne bekrefter at PM er obligatorisk for å kunne POST i prosjekt API. Han henviser til Daniel (som er på ferie) for å ta stilling til om dette kan (enkelt) endres.
Dermed blir det ingen lagring på 1. fane, i første runde. Vi kan evt. vurdere det når vi er tilbake frå ferie. Det er uansett ein enkel endring å legge til på eit seinare tidspunkt.

Reply:
Det er verken lov å opprette eller oppdatere et Prosjekt før man har lagt til en PM altså? 
Virker litt tungvidt om man skal endre tittel på et eksisterende Prosjekt (med PM), så ser man ikke noen lagre-knapp der, og man må skjønne selv at man må bla til neste tab for å lagre. 

I Registration-wizard er det ikke mulig å lagre før man har valgt type på resource-tab, så hvis man starter registrering med fil og lagrer på første tab får man feilmelding og vi viser feilmeldinger med hva som mangler. Tror det hadde vært en like god løsning her

Reply:
Supert. Da legg eg til Lagre knapp på 1. fane.